### PR TITLE
Added memory increase option to launch configurations

### PR DESCRIPTION
**Pull request by tudorlesan**
Fixes #https://github.com/CloudSlang/cloud-slang-content/issues/337 
Signed-off-by: Lesan Tudor tudor-andrei.lesan@hp.com

### CODE REVIEW

File: cloudslang-cli/pom.xml [hunk 1]
```diff
@@ -149,7 +149,7 @@
 							<id>cslang</id>
 						</program>
 					</programs>
-                    <extraJvmArguments>-Dpython.path=@BASEDIR@/python-lib</extraJvmArguments>
+                    <extraJvmArguments>-XX:PermSize=128m -XX:MaxPermSize=256m -Dpython.path=@BASEDIR@/python-lib</extraJvmArguments>
```
Comment (meirwah) [inline cloudslang-cli/pom.xml]: The issue you are trying to solve is in the CI (not regular usage of CLI)
It does not make sense to change CLI config , it makes more sense to try first to change CI (CircleCI) configuration....

**Comment by tudorlesan** [comment]
fixes CloudSlang/cloud-slang-content#337

File: cloudslang-cli/pom.xml (hunk 1 above)
Comment (tudorlesan) [inline cloudslang-cli/pom.xml]: oh ok, reverting it now

**Comment by orius123** [comment]
IIRC there is an issue of making a config file for the CLI, maybe it's a good place to start.

**Comment by tudorlesan** [comment]
CloudSlang/cloud-slang-content#343 fixes this instead

**Pull request closed without merging**